packages/amplify_datastore/test/amplify_datastore_delete_test.dart

```diff
@@ -35,37 +37,37 @@ void main() {
   });
 
 
-  test('delete returns 1 sucessful result (no query predicate)', () async {
+  test('delete with a valid model executes without an error ', () async {
+    var json = await getJsonFromFile('delete_api/request/instance_no_predicate.json');
 
-    var json = await getJsonFromFile('delete_api/response/1_deleted_result.json');
+    dataStoreChannel.setMockMethodCallHandler((MethodCall methodCall) async {});
 
-    dataStoreChannel.setMockMethodCallHandler((MethodCall methodCall) async {
-      return json;
-    });
+    Post instance = Post(title: json["model"]["title"], rating: json["model"]["rating"], created: DateTime.parse(json["model"]["created"]), id: json["model"]["id"]);
+    bool executed;
 
-    Post instance = Post(title: json["serializedData"]["title"], rating: json["serializedData"]["rating"], created: DateTime.parse(json["serializedData"]["created"]), id: json["id"]);
+    try {
+      await dataStore.delete(instance);
+      executed = true;
+    } catch (e) {
+      executed = false;
+    }
 
-    Post post = await dataStore.delete(instance);
-    expect(post.id, instance.id);
-    expect(post.title, instance.title);
-    expect(post.rating, instance.rating);
-    expect(post.created, instance.created);
+    expect(executed, true);
   });
 
-  test('delete returns 1 sucessful result (with query predicate)', () async {
+  test('delete with an invalid model executes with an error ', () async {
+    dataStoreChannel.setMockMethodCallHandler((MethodCall methodCall) async {});
 
-    var json = await getJsonFromFile('delete_api/response/1_deleted_result.json');
+    bool executed;
+    InvalidModel invalidModel = InvalidModel();
 
-    dataStoreChannel.setMockMethodCallHandler((MethodCall methodCall) async {
-      return json;
-    });
+    try {
```

Comment:
Ditto here. See if you can stave this out by using a tighter `test_api` predicate.

https://pub.dev/documentation/test_api/latest/test_api/test_api-library.html#properties

Reply:
I don't think this test is actually needed.